Templates/Empty-CSharp/function.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think setting it explicitly to the default value makes it more usable, without taking anything away from it